lexos/receivers/similarity_receiver.py

```diff
@@ -0,0 +1,24 @@
+from typing import NamedTuple
+from lexos.receivers.base_receiver import BaseReceiver
+
+
+class SimilarityOption(NamedTuple):
```

Comment:
Are we changing this to SimilarityFrontEndOption?

Reply:
Yes